Buildscripts/release-simulator.sh

```
@@ -12,6 +12,6 @@ target_path=$2
 mkdir -p $target_path
 
 cp version.txt $target_path
-cp $build_path/Firmware/FirmwareSim $target_path/
+cp $build_path/Tactility/Tactility $target_path/
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)